components/service-mgt/module-mgt/org.wso2.carbon.module.mgt/src/main/java/org/wso2/carbon/module/mgt/service/ModuleAdminService.java

```diff
@@ -73,15 +79,21 @@
     private static final String RAHAS_MODULE_NAME = "rahas";
     
     private PersistenceFactory pf;
+    
+    private Registry registry;
 
     public ModuleAdminService() throws Exception {
         this.axisConfig = getAxisConfig();
         pf = PersistenceFactory.getInstance(axisConfig);
+        registry = DataHolder.getRegistryService().getConfigSystemRegistry(
```

Comment:
Thanks Kishanthan,

With https://github.com/wso2/carbon-deployment/pull/135, moved the registry service access code to method level.